website/docs/terms/ddl.md

```diff
@@ -0,0 +1,126 @@
+---
+id: ddl
+title: DDL
+displayText: DDL  
+hoverSnippet: Data Definition Language (DDL) is a group of SQL statements that you can execute to manage database objects, including tables, views, and more.
+cta: staging
+---
+
+Data Definition Language (DDL) is a group of SQL statements that you can execute to manage database objects, including <Term id="table">tables</Term>, <Term id="view">views</Term>, and more. Using DDL statements, you can perform powerful commands in your database such as creating, modifying, and dropping objects. DDL commands are usually executed in a SQL browser or stored procedure.
+
+DDL is contrasted with <Term id="dml">Data Manipulation Language (DML)</Term> which is the SQL that is used to actually access and manipulate data in database objects. The majority of data analysts will rarely execute DDL commands and will do the majority of their work creating DML statements to model and analyze data. 
+
+:::note Note
+Data folks don’t typically write DDL [since dbt will do it for them](https://docs.getdbt.com/docs/about/overview#:~:text=dbt%20allows%20analysts%20avoid%20writing,dbt%20takes%20care%20of%20materialization.).
+:::
+
+To be honest, DDL is definitely some of the drier content that exists out there in the greater data world. However, because DDL commands are often uncompromising and should be used with caution, it’s incredibly important to understand how they work and when they should be used. We hope you can use this page to learn about the basics, strengths, and limitations of DDL statements.
+
+## Types of DDL Statements
+
+![Image of a One Does Not Simply Walk Into Mordor meme prompt that reads: One does not simply drop a table](/img/docs/terms/ddl/drop-a-table-meme.png)
```

Comment:
I'm the worst, but I think we should nix the image. The sizing is like just a bit off and we can live without it. Lmk if ppl agree!

Reply:
I personally like the flavor it adds, but agree that its a little large. That said, I don't think it's a show stopper. I can remove if you feel strongly about it, though

Reply:
Yeah I think we can nix it for now! I do like the flavor though lol

Reply:
Done!